starter-project/src/app/api/user/accounts/venmo/route.tsx

```diff
@@ -0,0 +1,25 @@
+import { NextResponse } from 'next/server';
+import { Bruinen } from '@bruinenco/bruinen-sdk';
+
+export async function GET(request: Request) {
+  const { searchParams } = new URL(request.url);
+  const accountId = searchParams.get('accountId');
+
+  if (!accountId) {
+    return new Response('Account ID not found', { status: 400 });
+  };
+
+  // Instantiate the Bruinen SDK: https://docs.bruinen.co/api-reference/bruinen-sdk
+  const bruinen = new Bruinen({
+    type: "apiKey",
+    apiKey: process.env.BRUINEN_SECRET || ""
```

Comment:
Instantiate outside of function, 

use `process.env.BRUINEN_SECRET as string` because `|| ""` isn't really correct since it will error out with an empty string.